libs/platform/src/lib/components/link/link.component.ts

```diff
@@ -0,0 +1,139 @@
+import {
+    AfterViewInit,
+    Component,
+    ChangeDetectionStrategy,
+    ChangeDetectorRef,
+    EventEmitter,
+    ElementRef,
+    Input,
+    OnInit,
+    Output,
+    ViewChild
+} from '@angular/core';
+import { coerceBooleanProperty } from '@angular/cdk/coercion';
+
+export type LinkType = 'standard' | 'emphasized';
+export type NavigationTarget = '_blank' | '_self' | '_parent' | '_top' | 'framename';
+const VALID_INPUT_TYPES = ['standard', 'emphasized'];
+const timeout: number = 1000;
+/** This Timeout should be reviewed */
+
+@Component({
+    selector: 'fdp-link',
+    templateUrl: './link.component.html',
+    styleUrls: ['./link.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class LinkComponent implements OnInit, AfterViewInit {
+    private _disabled: boolean = false;
+    private _inverted: boolean = false;
+    private _wrap: boolean = false;
+    isEmphasized: boolean = false;
+    tooltipVisibility: boolean = false;
+    isfocused: boolean = false;
+
+    /** Access child element, for checking link content*/
+    @ViewChild('link', { static: false }) anchor: ElementRef;
+
+    /** Id for the link */
+    @Input() id?: string;
```

Comment:
Please try to get aligned. When you start using decorators to be inlined with the property stay with it, or if you prefer to have it on new lines, stay with it but dont mix

Reply:
now change these into common signature like below
@Input()
id?: string